runtime/v2/sdk/src/main/java/com/walmartlabs/concord/runtime/v2/sdk/SensitiveData.java

```diff
@@ -51,5 +51,7 @@
 
     String[] keys() default {};
 
+    String[] paths() default {};
```

Comment:
As we discussed earlier, let's just use `keys`. I don't think that we will break anyone by treating dots `.` in `keys` as path separators.